tests/test_other.py

```diff
@@ -61,6 +61,19 @@ def decorated(self):
   return decorated
 
 
+def check_python3_version():
+  """emscripten requires at least python3.5 since python3.4 and below do not
```

Comment:
This is a weird docstring. It's a good necessary comment, but my understanding is that docstrings are for explaining a function and how it's used, whereas this explains historical context.
In particular I'd expect it to say something like "returns true if the system version of python3 is supported" or something